src/plugin/saml/Security/Authentication/AuthenticationSuccessListener.php

```diff
@@ -52,7 +52,12 @@ public function onAuthenticationSuccess(Request $request, TokenInterface $token)
         $user = $token->getUser();
 
         // apply IDP config
-        $idpEntityId = $request->getSession()->get('authIdp');
+        $sessions = $request->getSession()->get('samlsso')->getSsoSessions();
+        if (empty($sessions)) {
+            return null;
```

Comment:
Method must return a response. It will throw an error like it. You should `return parent::onAuthenticationSuccess($request, $token);`.